### PR TITLE
Remove the redundant parentheses and format the code

### PR DESCRIPTION
**Pull request by wackxu**
**What this PR does / why we need it**:

Remove the redundant parentheses and format the code

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by wackxu** [comment]
@dmmcquay @freeformz Can you help review this pr? thanks!

File: cmd/kubeadm/test/cmd/completion_test.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func TestCmdCompletion(t *testing.T) {
 				rt.args,
 				actual,
 				rt.expected,
-				(actual == nil),
```
Comment (CaoShuFeng) [inline cmd/kubeadm/test/cmd/completion_test.go]: What's the motivation of this change?
Is there any document or standard says it's better to remove ths parentheses?

IMO, when the `hack/verify-gofmt.sh` says the format is OK, then the format is OK.

Comment (wackxu) [reply cmd/kubeadm/test/cmd/completion_test.go]: When I read the code , there is a  weak warning " Report redundant parentheses expressions" in the Gogland ide. And verify-gofmt.sh can automatically fix the majority of mechanical style issues but not the all issues.In the [https://golang.org/doc/effective_go.html#formatting](url), if the operator precedence hierarchy is shorter and clearer,the parentheses is not necessary.In a word,the format of this Pr is better but not required.

Comment (CaoShuFeng) [reply cmd/kubeadm/test/cmd/completion_test.go]: Seems good to me, thanks.

**Review by dmmcquay — APPROVED** [review]

**Review by dmmcquay — CHANGES_REQUESTED** [review]
I can agree that the parentheses are redundant.  If we are going to change this in these files, I would suggest we go throughout the kubeadm code base and change it (for example: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/endpoint_test.go#L109) and have it uniform in all places.

**Comment by wackxu** [comment]
@dmmcquay Thanks for your suggest and I will check all the file in the kubeadm and fix it.

**Comment by wackxu** [comment]
ping @dmmcquay

**Review by dmmcquay — APPROVED** [review]
lgtm
